src/main/scala/org/bblfsh/client/libuast/org_bblfsh_client_libuast_Libuast.c

```diff
@@ -22,6 +26,8 @@ JNIEXPORT jobject JNICALL Java_org_bblfsh_client_libuast_Libuast_filter
   if ((*env)->MonitorEnter(env, self) != JNI_OK)
     goto exit;
 
+  initAllocList(&allocList, ALLOCLIST_SIZE);
```

Comment:
Minor: Initial vector size is probably better left as an implementation detail.

Reply:
:+1: